src/pip/_internal/wheel.py

```diff
@@ -885,7 +892,10 @@ def _build_one_inside_env(self, req, output_dir, python_tag=None):
                 wheel_name = os.path.basename(wheel_path)
                 dest_path = os.path.join(output_dir, wheel_name)
                 try:
+                    wheel_hash, _ = hash_file(wheel_path)
```

Comment:
I think it would be helpful for readers of the code to give a name to the second variable in the return value. You could add a code comment above the line saying, "We don't need to use the second value." if you would like to communicate that.

Reply:
`_length` should do, or also output the file size :)

Reply:
I have put the length into the output string with a8e382b, so this value is now used